packages/ui/src/vue-book/book-main.ts

```diff
@@ -45,6 +45,12 @@ app.use(VuesticPluginsWithoutComponents, {
     ...colorsPresets.default,
     banana: '#d0f55d',
   },
+  classHelpers: [
```

Comment:
You can create color-css-helpers-updater.demo.vue and have simple demo here. It would be helpful for manual testing (and cypress later).
```vue
<div class="demo--banana"> I'm yellow </div>
```